agent/bench-scripts/postprocess/uperf-prepare-xmlfile

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: t; sh-basic-offset: 8; sh-indentation: 8; sh-indent-for-case-alt: + -*-
+#echo "<profile name=\"protocol:\$protocol-test_type:\$test_type-wsize:\$wsize-threads:\$threads\">"
```

Comment:
Is this for debugging?

Reply:
No, it's just old code that needs to be removed.